README.md

```diff
@@ -67,3 +67,16 @@ php cheat.php token2
 2. To build: `docker build . -t steamdb/saliencheat`
 3. To run: `docker run -it --init --rm -e TOKEN=<32 character token from gettoken url> steamdb/saliencheat`
 4. To stop running, Ctrl+C
+
+## Heroku
+1. Fork this repository on your GitHub account.
```

Comment:
Is the fork really necessary?

Reply:
In order for the connection to happen, Heroku only pulls from owned or membered repos under the user. (Such is the Oauth scopes method they use.)

Reply:
Yep, repo admin access required.

Reply:
Believe it or not this actually isn't required if you set up a one-click heroku deploy button like I have in PR #190

Reply:
Nice job @JourneyOver!